src/frontend/next/src/pages/404.tsx

```diff
@@ -0,0 +1,12 @@
+import { FC } from 'react';
+
+const MissingPage: FC = () => {
+  return (
+    <div>
+      <h1>404</h1>
+      <p>We could not find what you were looking for</p>
+    </div>
+  );
+};
+
+export default MissingPage;
```

Comment:
Do you need this code part? I want to know the difference from "_error.tsx". Can you add an explanation?